docs/dev/releasing.rst

```diff
@@ -305,15 +305,19 @@ Install the Python requirements for the driver tools::
    $ pip install -r $CDRIVER_TOOLS/requirements.txt
 
 
-**For Patch Releases**: Check Consistency with the Jira Release
-***************************************************************
+Check Consistency with the Jira Release
+***************************************
```

Comment:
Tweak the item in the checklist at the top of this page to match this section title.